doc/host_function.md

```diff
@@ -223,4 +223,4 @@ From our mechanism, `Expect<T>` declared in `include/common/errcode.h` is used a
 
 ### Forcing Termination
 
-SSVM provides a method for terminating WASM execution in host functions. Developers can return `ErrCode::Terminated` to trigger the forcing termination of the current execution and the execution result will be set as successful.
+WasmEdge provides a method for terminating WASM execution in host functions. Developers can return `ErrCode::Terminated` to trigger the forcing termination of the current execution and the execution result will be set as successful.
```

Comment:
So... It will not trigger a forcing termination now?

Reply:
It will trigger a forcing termination, and return error with `ErrCode::Terminated`. The caller should recognize for the `Success` or the `Terminated`.